compiler-rt/lib/scudo/standalone/combined.h

```diff
@@ -1281,7 +1281,8 @@ class Allocator {
 
   void storeSecondaryAllocationStackMaybe(const Options &Options, void *Ptr,
                                           uptr Size) {
-    if (!UNLIKELY(Options.get(OptionBit::TrackAllocationStacks)))
+    if (!UNLIKELY(Options.get(OptionBit::TrackAllocationStacks)) ||
+        RawRingBuffer == nullptr)
```

Comment:
Can we use `RawRingBuffer` as a switch here? Or do we have the scenario that we may want to turn on/off the stack storing randomly?

Reply:
I would rather not change the external behavior without good reason. It is currently certainly possible to turn it of and back on.

Reply:
Just reviewed the logic, I may think that we may want to support turning on/off TrackAllocationStacks by env variable so that we won't have two ways to control this function. Same as above, no strong opinion here.